subsys/bluetooth/services/Kconfig.ots

```diff
@@ -0,0 +1,9 @@
+# Bluetooth GATT Object Transfer service
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_GATT_OTS
+	bool "Object Transfer Service libraries"
```

Comment:
Let mark this as experimental so we can fix this latter if needed.

Reply:
This is marked as experimental in the Kconfig that is located in OTS library. I am planning to merge both files into one and put the tag on the top-level configuration (that is BT_OTS)

Reply:
Now, the whole configuration is placed in one file services/ots/Kconfig with experimental tag attached to the BT_OTS definition